packages/babel-plugin/vitest.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    globals: true,
```

Comment:
only in @kuma-ui/babel-plugin, `globals` is set to `true` for babel-plugin-tester behavior.